### PR TITLE
chore: expose `NormalizedWheelEvent`

### PR DESCRIPTION
**Pull request by holazz**

### CODE REVIEW

**Comment by holazz** [comment]
@sxzz PTAL again 🙏

**Comment by sxzz** [comment]
What's your purpose of exporting `NormalizedWheelEvent`?

File: index.d.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
+export interface NormalizedWheelEvent {
```
Comment (sxzz) [inline index.d.ts:1]: exporting typing will make `declare module` fail

**Comment by holazz** [comment]
I used it here:

```ts
import type { NormalizedWheelEvent } from 'normalize-wheel-es'

const mousewheel = function (
  element: HTMLElement,
  callback: (e: WheelEvent, normalized: NormalizedWheelEvent) => void
) {
  ...
}
```

File: index.d.ts (hunk 1 above)
Comment (holazz) [reply index.d.ts:1]: sorry, I didn't take that into consideration.

Comment (sxzz) [reply index.d.ts:1]: I fixed it in the new commit.

Comment (holazz) [reply index.d.ts:1]: Thanks, This also works:

```ts
callback: (e: WheelEvent, normalized: ReturnType<typeof normalizeWheel>) => void
```

**Pull request closed without merging**